### DISCUSSION TITLE
Compiling Anura3D in Microsoft Visual Studio Community 2022

### DISCUSSION THREAD

**Discussion by DanBrox** [Anura3D/Anura3D_OpenSource#251] [Q&A]
Attempting to follow the Anura3D download and compile in visual studio instructions.  Downloaded oneAPI Base Toolkit and HPC Toolkit, but getting the attached 'error #7002: Error in opening the compiled module file.  Check INCLUDE paths.   [MODERRORHANDLER]		C:\Program Files (x86)\Anura3D_OpenSource-main\src\ErrorHandler.for	284'.  Suggestions?
![BeginnerQuestion](https://github.com/user-attachments/assets/ab07e3e4-e743-4aee-92d2-c05f35a541e7)

**Comment by CH-max92** [comment]
The content of this post (https://github.com/Anura3D/Anura3D_OpenSource/discussions/235) in the discussion may be helpful to you.
Making good use of the search function in the community will help you get started with the code.
